apps/forwarder-ovs/forwarder.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: forwarder-ovs
+  labels:
+    app: forwarder-ovs
+spec:
+  selector:
+    matchLabels:
+      app: forwarder-ovs
+  template:
+    metadata:
+      labels:
+        app: forwarder-ovs
+    spec:
+      hostPID: true
+      hostNetwork: true
+      containers:
+        - image: networkservicemeshci/cmd-forwarder-ovs:latest
```

Comment:
Typically we bind to particular version of a container (and they get updated from downstream).  Could that be done here?

Reply:
done as above.